be/src/common/config.h

```diff
@@ -1049,4 +1049,6 @@ CONF_mInt32(get_txn_status_internal_sec, "30");
 
 CONF_mBool(dump_metrics_with_bvar, "true");
 
+CONF_mBool(enable_drop_tablet_if_unfinished_txn, "true");
```

Comment:
as we discussed, the default value should be false?

Reply:
It's better to not change the current behavior. The balance and drop conflict is less than the drop table itself.